libs/community/langchain_community/vectorstores/bigquery_vector_search.py

```diff
@@ -88,10 +89,15 @@ def __init__(
                 to use. Defaults to None.
         """
         try:
+            from google.api_core import client_info
             from google.cloud import bigquery
 
```

Comment:
can we reuse this method maybe?
https://github.com/langchain-ai/langchain/blob/37062549f9704765b147330b11988d8f5151f301/libs/partners/google-vertexai/langchain_google_vertexai/_utils.py#L60

Reply:
Thanks. Done.